src/utils/rules-proxy.js

```diff
@@ -59,13 +59,8 @@ const getCountry = function () {
 
 const createRewriter = async function ({ distDir, projectDir, jwtSecret, jwtRoleClaim, configPath }) {
   let matcher = null
-  const configFiles = [
-    ...new Set(
-      [path.resolve(distDir, '_redirects'), path.resolve(projectDir, '_redirects'), configPath].filter(
-        (configFile) => configFile && configFile !== projectDir,
-      ),
-    ),
-  ]
+  const redirectsFiles = [...new Set([path.resolve(distDir, '_redirects'), path.resolve(projectDir, '_redirects')])]
```

Comment:
In production, I believe only the publish directory (named `distDir` here) can be used for `_redirects`. That's at least what my manual test + the documentation website indicate.
`projectDir` is the build directory. do we have a reason why `netlify dev` allows `_redirects` in that directory, even though it is not valid in production?